### PR TITLE
Merge dev to master for 1.0.0 release

### PR DESCRIPTION
**Pull request by cholmes**
**Related Issue(s):** https://github.com/radiantearth/stac-spec/milestone/23

**Proposed Changes:**

1. Merge dev to master for 1.0.0 release

**PR Checklist:**

- [ ] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [ ] This PR has **no** breaking changes.
- [ ] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md)
      **or** a CHANGELOG entry is not required.
- [ ] This PR affects the [STAC API spec](https://github.com/radiantearth/stac-api-spec),
      and I have opened issue/PR #XXX to track the change.

### CODE REVIEW

**Review by jbants — APPROVED** [review]

**Review by matthewhanson — APPROVED** [review]

**Review by jbants — APPROVED** [review]

**Comment by m-mohr** [comment]
There's no way back once someone clicks the mighty Merge button... :-)

**Review by m-mohr — APPROVED** [review]

**Pull request merged**
